firefox-desktop/views/mobile_kpis.view.lkml

```diff
@@ -0,0 +1,78 @@
+# Mapping of forecast `product` column to mobile_usage_2021 `canonical_app_name`` (for join)
+# Focus Android == Firefox Focus for Android
+# Focus iOS == Firefox Focus for iOS
+# Firefox Android == Firefox for Android (Fenix)
+# Firefox Android == Firefox for Android (Fennec)
+# Firefox iOS == Firefox for iOS
+
+view: mobile_kpis {
+  derived_table: {
+    sql: SELECT
+            TIMESTAMP(submission_date) as date
+            , CASE WHEN canonical_app_name = "Firefox Focus for Android" THEN "Focus Android"
```

Comment:
What do I with the Fenix-Fennec combination?

Reply:
Ooph, good point. I want @jklukas to chime in here because he's spent a lot of time trying to keep these names organized in BQ.

Reply:
I suppose I just remove the paranthesized bit from the canonical_app_name so both values are just e.g.  Firefox for Android ~(Fenix)~

Reply:
I would like to change the logic here to match based on `app_name` rather than `canonical_app_name`. `canonical_app_name` is intended only for final display in things like dashboards (like this one... do you think it's reasonable to leave names like "Firefox Focus for iOS" intact here?), and my intention would be that `app_name` is short and stable and can be used for in-query logic that needs to distinguish between apps.

Reply:
I would ideally like the display name for the combined forecast to be "Firefox for Android (Fennec + Fenix)" to be super clear.

Reply:
OK I'll do this